File: server/FlightLog-server/src/main/java/bitc/fullstack503/flightlog/flightlogserver/dto/flightInfoDTO.java
package bitc.fullstack503.flightlog.flightlogserver.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class flightInfoDTO {
  @JsonProperty("flightId")
  private int flightId;

  @JsonProperty("flightInfoAirline")
  private String flightInfoAirline;

  @JsonProperty("flightInfoArrivalCity")
  private String flightInfoArrivalCity;

  @JsonProperty("flightInfoArrivalTime")
  private String flightInfoArrivalTime;

  @JsonProperty("flightInfoEddate")
  private String flightInfoEddate;

  @JsonProperty("flightInfoAirlineNum")
  private String flightInfoAirlineNum;

  @JsonProperty("flightInfoStartTime")
  private String flightInfoStartTime;

  @JsonProperty("flightInfoStartDate")
  private String flightInfoStartDate;

  @JsonProperty("flightInfoStartCity")
  private String flightInfoStartCity;

  @JsonProperty("flightMon")
  private String flightMon;

  @JsonProperty("flightTue")
  private String flightTue;

  @JsonProperty("flightWed")
  private String flightWed;

  @JsonProperty("flightThu")
  private String flightThu;

  @JsonProperty("flightFri")
  private String flightFri;

  @JsonProperty("flightSat")
  private String flightSat;

  @JsonProperty("flightSun")
  private String flightSun;
}

File: app/app/src/main/java/bitc/fullstack/FlightLog/dto/flightInfoDTO.kt
package bitc.fullstack.FlightLog.dto

import com.google.gson.annotations.SerializedName

data class flightInfoDTO(
  @SerializedName("flightId")
  var flightId: Int,

  @SerializedName("flightInfoAirline")
  var flightInfoAirline: String,

  @SerializedName("flightInfoArrivalCity")
  var flightInfoArrivalCity: String,

  @SerializedName("flightInfoArrivalTime")
  var flightInfoArrivalTime: String,

  @SerializedName("flightInfoEddate")
  var flightInfoEddate: String,

  @SerializedName("flightInfoAirlineNum")
  var flightInfoAirlineNum: String,

  @SerializedName("flightInfoStartTime")
  var flightInfoStartTime: String,

  @SerializedName("flightInfoStartDate")
  var flightInfoStartDate: String,

  @SerializedName("flightInfoStartCity")
  var flightInfoStartCity: String,

  @SerializedName("flightMon")
  var flightMon: String,

  @SerializedName("flightTue")
  var flightTue: String,

  @SerializedName("flightWed")
  var flightWed: String,

  @SerializedName("flightThu")
  var flightThu: String,

  @SerializedName("flightFri")
  var flightFri: String,

  @SerializedName("flightSat")
  var flightSat: String,

  @SerializedName("flightSun")
  var flightSun: String
)